firebase-template.js

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import {initializeApp} from "firebase/app";
+import {getAnalytics} from "firebase/analytics";
+import {getAuth} from "firebase/auth";
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+const firebaseConfig = {
```

Comment:
can it be placed here?

Reply:
is it safe?

Reply:
I think it is. It is very stupid way of injecting the env variables to our application. I tried really too many ways of doing that but nothing worked. We have this template config with mocked `ENV_VAR...` strings which are replaced by the real env variables by generate-firebase bash script before running the actual application. (if you will run the appliction locally there will be generated file firebase.js with the actual keys